app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -17,6 +17,7 @@ import org.mozilla.tv.firefox.telemetry.TelemetryIntegration
 import org.mozilla.tv.firefox.utils.UrlUtils
 
 data class ValidatedIntentData(val url: String, val source: Session.Source)
+private const val ACTIVE_EXPERIMENTS_KEY = "activeExperiments"
```

Comment:
nit: constants should be defined about classes. Also, imo classes & values should generally have newlines between them.

Reply:
I'm reading this as "above classes" and "add a newline"